static/favicon_package_v0.16/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
+    "short_name": "",
+    "icons": [
+        {
+            "src": "/android-chrome-192x192.png",
+            "sizes": "192x192",
+            "type": "image/png"
+        },
+        {
+            "src": "/android-chrome-512x512.png",
+            "sizes": "512x512",
+            "type": "image/png"
+        }
+    ],
+    "theme_color": "#ffffff",
```

Comment:
Wondering if `theme_color` should be what's int he `_theme.js` default prop of #8E1CFA or one of those instead of white?

Reply:
I think that would depend on if it overwrites #8E1CFA

Reply:
We wouldn't bring in values from `_theme.js` just use the values from it as the content values